installer/x86_64/install.sh

```diff
@@ -569,7 +622,15 @@ if [ "$install_env" = "onie" ]; then
     mv $onie_initrd_tmp/tmp/onie-support*.tar.bz2 $demo_mnt/$image_dir/
 
     if [ "$firmware" = "uefi" ] ; then
-        demo_install_uefi_grub "$demo_mnt" "$blk_dev"
+        secure_boot_state=$(mokutil --sb-state)
```

Comment:
please note. I have had an issue with the latest master branch when i built for broadcom. it failed at this line as it could not find mokutil to check secureboot state
https://github.com/sonic-net/sonic-buildimage/issues/14316

Reply:
Are you using it on 202205? Looks like https://github.com/sonic-net/sonic-buildimage/issues/14316 is still open.
Once the issue is resolved I ll cherrypick them too. Thanks for letting me know.

Reply:
I was using it on master. 
Seems my onie env doesnt have the command and the hardware model hasnt had an official onie update. 

Would it make sense to have it use the command from the mounted image instead of from the onie env?

Reply:
I already suggested a solution to @ITJamie , Im pending to testing it with a device with old onie version.
I will add you as a reviewer as well.

Reply:
Hi David 
@ITJamie mentioned he will not use 202205. I can cherry-pick the specific fix later.
Can you review this PR?

Reply:
sure, you have differnt options:
1. install an update ONIE vertion than 2021.11
2. install manually in ONIE the mokutil command.
3. build the image including this fix: https://github.com/sonic-net/sonic-buildimage/pull/14429
Pls let me know if it solve the problem, thanks

Reply:
Thanks David!

Reply:
Backported this change.